マニュアル.md

```diff
@@ -1,51 +1,203 @@
+- **Updates per second:** デバイスに動作コマンドを送信する頻度を指定します。BLEデバイスは通常、1秒間に約10~20個のコマンドを処理できます。
+
+オナホ設定:
+- **Max Strokes (per minute):** 100%の長さで上下させることが可能な最高速度です。
+- **Stroke Zone / Slow:** 動きをスローにしたときの上下運動の長さです。0%が最小、100%が最大です。
+- **Smooth Stroking:** 上下運動の動きからロボっぽさを減らしますが、全てのオナホが対応しているわけではありません。HandyとOSR2デバイスで上手く機能することが分かっています。デフォルトではオフになっています。
+
+バイブ設定:
+- **Intensity Range:** このデバイスで可能な最小~最大の振動の強さを設定します。0%=振動無し、100%=振動最大。
+- **Vibration Pattern:** 振動のパターンを指定します。利用可能なパターンはSin波、三角波、鋸波、パルス波、変化なし、カスタムパターンです。
+- **Custum Pattern:** Vibration PatternがCustomに設定されている場合に使用できますスライダーを使用して、振動強度のカーブを設定することができます。
+
+圧力設定:
+- **Pressure Range:** このデバイスが許容する最小および最大の圧力を設定します。単位は(%)です。
+- **Pressure Update Interval (seconds):** デバイスが圧力を変更するのに掛かる時間です。単位は(秒)。デフォルトは5秒です。
+
+以下の設定も行うといいかもしれません:
+- **Save device assignments:** 有効にすると、すべてのデバイス「3Pの役割」と「身体の部位属性」が共有できます。デフォルトでは無効になっています。
+
+### Intiface設定
+- **Intiface CLI location:** Intiface CLIの実行ファイルがあるパスです。LoveMachineはゲームが起動すると、このプログラムを起動しようとします。
+- **WebSocket host:** 実行中のIntifaceホストのURLです。リモートマシン上で実行されていない限り、`ws://localhost`でなければなりません。
+- **WebSocket port:** Intifaceが解放しているポートです。通常は`12345`です。
+
+### 停止ボタン設定
+万が一何かが起こった時、または激しいセックスが始まってしまった時、あなたを守るために停止ボタンをご用意しました。デフォルトではスペースキーを押すことで全てのデバイスが直ちに停止します。
+
+- **Emergency Stop Key Binding:** 停止ボタンを作動させるためのキー割り当てを設定します。(デフォルトではスペースキー)
+- **Resume Key Binding:** 停止を解除するためのキー割り当てを設定します。(デフォルトではF8)
+
+### オナホ設定
+- **Stroke Length Realism:** 上下運動の幅をアニメーションとどの程度一致させるかを設定します。0%ではデバイスが利用可能な長さをフル活用します。100%ではゲーム内のアニメーションの長さにできるだけ合わせて動きます。
+
+- **Hard Sex Intensity:** 激しいセックスアニメーション中にデバイスが動く強さを調整します。100%にすると0%のときより2倍早く動きます(少なくともHandyでは)。
+LoveMachineの使用により発生するいかなる損害についても、開発者は一切の責任を負いかねます。
+
+- **Orgasm Depth:** 絶頂時のデバイスの動きの深さを設定します。
+- **Orgasm Shaking Frequency:** 絶頂中に1秒間で何回上下運動を行うかを設定します。
+
+### 回転設定
+- **Rotation Speed Ratio:** 回転の速度を設定します。0%は無回転、100%全速回転です。デフォルトは50%です。
+- **Rotation Direction Change Chance:** 回転方向が変化する確率を設定します。デフォルトは30%です。
+
+### 圧力設定
+- **Enable Pressure Control:** このデバイスの圧力機能を使用するかどうかを設定します。デフォルトでは無効になっています。
+- **Pressure Mode:** 圧力の設定方法を決定します。
+  - **Cycle:** 一定時間かけて徐々に圧力を高め、最後に開放します。
+  - **Stroke Length:** 圧力をかける面積です。圧力が強くなります。
+  - **Stroke Speed:** 圧力がかかる速さです。圧力が強くなります。
+- **Pressure Cycle Length (seconds):** 圧力設定がCycleに設定されている場合、圧力が強まってから解放されるまでの時間間隔を設定します。単位は(秒)です。
+
+
 * A10ピストンSA
 * [The Handy](https://www.thehandy.com/?ref=saucekebenfield&utm_source=saucekebenfield&utm_medium=affiliate&utm_campaign=The+Handy+Affiliate+program)
 * KIIROO KEON
 * Xboxゲームパッド（振動）
 * [そしてこのリストにあるデバイスほぼすべて](https://iostindex.com/?filter0ButtplugSupport=4)
 
+## ご協力してくれる方へ
+対応してほしいゲームタイトルがあれば是非ご連絡ください。新しいゲームタイトルに対応するのは比較的簡単で、ほとんどコーディングをしなくてもよいのです。参考としてPlayHomeの実装をご覧ください。また、対応してほしいデバイスのPRも大歓迎です。
 
-## 使い方
+このMODは無料ですが、寄付を受け付けています。私の開発を応援してくださる方は、[こちら]からよろしくお願いします。
+### 開発者
+Sauceke   •   nhydock   •   hogefugamoga   •   RPKU

```

Comment:
You are contributing to the code base, so your name should go here.

Reply:
Thank you very much. I am afraid I will add my name there.